model/quic-socket-base.cc

```diff
@@ -1907,6 +1937,16 @@ QuicSocketBase::SetConnectionId (uint64_t connectionId)
   m_connectionId = connectionId;
 }
 
+void
+QuicSocketBase::InitializeScheduling ()
```

Comment:
There was a question on this in the previous review - can you check?

"Why are you calling this explicitly from L4 and not in the constructor for QuicSocketBase?"